tests/util_tests/test_process_file.py

```diff
@@ -16,7 +17,7 @@ def db():
 
     yield arlo_server.db
 
-    arlo_server.db.session.commit()
+    arlo_server.db.session.rollback()
```

Comment:
This just mapped better to the behavior we actually want for a test. Just because a test got its db session wedged doesn't necessarily mean we should also throw _again_ on teardown.